library/aes.c

```diff
@@ -33,6 +33,29 @@
 #include "mbedtls/platform.h"
 #include "mbedtls/platform_util.h"
 #include "mbedtls/error.h"
+
+#if defined(MBEDTLS_HAVE_ASM) && \
+    defined(__aarch64__) && !defined(MBEDTLS_HAVE_ARM64)
```

Comment:
I don't understand the use of `defined(__aarch64__) && !defined(MBEDTLS_HAVE_ARM64)`

`MBEDTLS_HAVE_ARM64` is set in `aesce.h` when `defined(__aarch64__) || defined(_M_ARM64) || defined(_M_ARM64EC)`

So this means "we're compiling for `__aarch64__` but haven't `#include`d `aesce.h`"?

What am I missing?

Reply:
Thanks. I forget remove it when removing `#define MBEDTLS_HAVE_ARM64`